c/json.c

```diff
@@ -164,7 +178,16 @@ void jsonWriteBufferInternal(jsonPrinter *p, char *text, int len) {
     p->customWrite(p, text, len);
   } else {
     while (bytesWritten < len) {
-#ifndef METTLE
+#if defined(__ZOWE_OS_WINDOWS)
+      int newWriteReturn = -666; /* send(p->fd,text+bytesWritten,len-bytesWritten,0); */
+      if (p->fd == _fileno(stdout)){
+        newWriteReturn = fwrite(text, 1, len, stdout);
+      } else if (p->fd == _fileno(stderr)){
+        newWriteReturn = fwrite(text, 1, len, stderr);
+      } else {
+        printf("JOE WINDOWS FAKE SOCKET WRITE bad case\n");
```

Comment:
Should we have a proper message, like those `JSONERROR` and without personal info? :)

Reply:
fixed in next commit.